onnxruntime/core/providers/rocm/tunable/gemm_tunable.cuh

```diff
@@ -37,6 +37,16 @@ class GemmTunableOp : public tunable::TunableOp<GemmParams<T>> {
  public:
   GemmTunableOp() {
     this->ops_.emplace_back(RocBlasGemmOp<T>);
+
+  #if defined USE_ROCBLAS_EXTENSION_API
```

Comment:
style: `#ifdef` or `#if defined(...)`

Also please format the code here.

Reply:
Done!